next/src/components/_global/nav/nav.tsx

```diff
@@ -116,42 +189,46 @@ export function Hamburger({
   setToggleNav: (toggleNav: boolean) => void;
 }) {
   return (
-    <svg
-      xmlns='http://www.w3.org/2000/svg'
-      width='62'
-      height='61'
-      fill='none'
-      viewBox='0 0 62 61'
+    <button
       onClick={() => setToggleNav(!toggleNav)}
       className={styles.exit}
     >
-      <path
-        stroke='#163C3E'
-        strokeLinecap='round'
-        strokeWidth='4'
-        d='M10.917 17.792h7.625m33.042 0H28.709M51.583 43.208h-7.625m-33.042 0h22.875M10.917 30.5h40.667'
-      ></path>
-    </svg>
+      <svg
+        xmlns='http://www.w3.org/2000/svg'
+        width='62'
+        height='61'
+        fill='none'
+        viewBox='0 0 62 61'
+      >
+        <path
+          stroke='#163C3E'
+          strokeLinecap='round'
+          strokeWidth='4'
+          d='M10.917 17.792h7.625m33.042 0H28.709M51.583 43.208h-7.625m-33.042 0h22.875M10.917 30.5h40.667'
+        ></path>
+      </svg>
+    </button>
   );
 }
 
 export function Exit({ toggleNav, setToggleNav }: { toggleNav: boolean; setToggleNav: (toggleNav: boolean) => void }) {
   return (
-    <svg
-      xmlns='http://www.w3.org/2000/svg'
-      width='42'
-      height='42'
-      fill='none'
-      viewBox='0 0 42 42'
-      onClick={() => setToggleNav(!toggleNav)}
-    >
-      <path
-        stroke='#163C3E'
-        strokeLinecap='round'
-        strokeWidth='1.5'
-        d='M25.375 16.625l-8.75 8.75m0-8.75l8.75 8.75M38.5 21c0 8.25 0 12.374-2.563 14.937C33.374 38.5 29.25 38.5 21 38.5s-12.374 0-14.937-2.563C3.5 33.374 3.5 29.25 3.5 21s0-12.374 2.563-14.937C8.626 3.5 12.75 3.5 21 3.5s12.374 0 14.937 2.563c1.704 1.704 2.275 4.099 2.467 7.937'
-      ></path>
-    </svg>
+    <button onClick={() => setToggleNav(!toggleNav)}>
```

Comment:
You should specify the `<button>` with `onClick` inline, straight into the component, and inside that specify e.g. `<CloseIcon />` which you declare under the `default export {Component}`